Pyrado/pyrado/environments/mujoco/openai_humanoid.py

```diff
@@ -0,0 +1,211 @@
+        min_init_state = np.concatenate([min_init_qpos, min_init_qvel, min_init_rest]).ravel()
+        max_init_state = np.concatenate([max_init_qpos, max_init_qvel, max_init_rest]).ravel()
+        self._init_space = BoxSpace(min_init_state, max_init_state)
+
+        self.camera_config = dict(distance=5.0)
+
+    @property
+    def state_space(self) -> Space:
+        state_shape = np.concatenate(
+            [
+                self.sim.data.qpos.flat,
+                self.sim.data.qvel.flat,
+                self.sim.data.cinert.flat,
+                self.sim.data.cvel.flat,
+                self.sim.data.qfrc_actuator.flat,
+                self.sim.data.cfrc_ext.flat,
+            ]
+        ).shape
+        return BoxSpace(-pyrado.inf, pyrado.inf, shape=state_shape)
+
+    @property
+    def obs_space(self) -> Space:
+        obs_shape = self.observe(self.state_space.bound_up).shape
+        return BoxSpace(-pyrado.inf, pyrado.inf, shape=obs_shape)
+
+    @property
+    def act_space(self) -> Space:
+        act_bounds = self.model.actuator_ctrlrange.copy().T
+        return BoxSpace(
+            *act_bounds,
+            labels=[
+                "abdomen_y",
+                "abdomen_z",
+                "abdomen_x",
+                "right_hip_x",
+                "right_hip_z",
+                "right_hip_y",
+                "right_knee",
+                "left_hip_x",
+                "left_hip_z",
+                "left_hip_y",
+                "left_knee",
+                "right_shoulder1",
+                "right_shoulder2",
+                "right_elbow",
+                "left_shoulder1",
+                "left_shoulder2",
+                "left_elbow",
+            ],
+        )
+
+    @classmethod
+    def get_nominal_domain_param(cls) -> dict:
+        return dict(
+            reset_noise_halfspan=1e-2,
+            gravity=9.81,
+            sliding_friction=1,
+            torsional_friction=0.005,
+            rolling_friction=0.0001,
+            density=1000,  # scales linearly with the mass
+            wind_x=0,
+            wind_y=0,
+            wind_z=0,
+        )
+
+    def _create_task(self, task_args: dict) -> Task:
+        # Define the task including the reward function
+        if "contact_cost_weight" not in task_args:
+            task_args["contact_cost_weight"] = 5e-7
+        if "ctrl_cost_weight" not in task_args:
+            task_args["ctrl_cost_weight"] = 0.1
+        if "forward_reward_weight" not in task_args:
+            task_args["forward_reward_weight"] = 1.25
+        if "healthy_reward" not in task_args:
+            task_args["healthy_reward"] = 5.0
+        if "terminate_when_unhealthy" not in task_args:
+            task_args["terminate_when_unhealthy"] = True
+        if "healthy_z_range" not in task_args:
+            task_args["healthy_z_range"] = (1.0, 2.0)
+        if "contact_cost_range" not in task_args:
+            task_args["contact_cost_range"] = (-np.inf, 10.0)
```

Comment:
I would suggest to move these default values to the constructor of the reward function